source/gui/settingsDialogs.py

```diff
@@ -3408,6 +3414,40 @@ def onRemove(self, evt):
 		scriptInfo.gestures.remove(gesture)
 		self.tree.SetFocus()
 
+	def onReset(self, evt):
+		if gui.messageBox(
+			# Translators: A prompt for comfirmation to reset all gestures in the Input Gestures dialog.
+			_("""Are you sure you want to reset all gestures to their factory defaults?
+			
+			All past and present user defined gestures will be lost.

```

Comment:
I understand what you're saying with "past and present ", but I don't think it quite conveys the weight you are trying to imply.

```suggestion
			All of your user defined gestures, whether previously set or defined during this session, will be lost.
```

Reply:
Thanks